state/state.go

```diff
@@ -12,10 +12,11 @@ import (
 
 	"github.com/roboll/helmfile/helmexec"
 
-	yaml "gopkg.in/yaml.v1"
```

Comment:
My IDE runs go fmt so there are a few formatting changes not directly related to the PR. Not sure if you have strong feelings on that.